src/sidebar/components/ThreadCard.tsx

```diff
@@ -53,36 +56,41 @@ function ThreadCard({ frameSync, thread }: ThreadCardProps) {
 
   // Handle requests to give this thread keyboard focus.
   const focusRequest = store.annotationFocusRequest();
-  const cardRef = useRef<HTMLElement | null>(null);
   useEffect(() => {
-    if (focusRequest !== thread.id || !cardRef.current) {
+    if (focusRequest !== thread.id || !threadRef.current) {
       return;
     }
-    cardRef.current.focus();
+    threadRef.current.focus();
     store.clearAnnotationFocusRequest();
   }, [focusRequest, store, thread.id]);
 
   return (
-    /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
-    <Card
-      active={isHovered}
-      classes="cursor-pointer focus-visible-ring theme-clean:border-none"
-      data-testid="thread-card"
-      elementRef={cardRef}
-      tabIndex={-1}
+    <ButtonBase
+      unstyled
+      classes="focus-visible-ring text-left w-full"
+      elementRef={threadRef}
       onClick={e => {
         // Prevent click events intended for another action from
         // triggering a page scroll.
-        if (!isFromButtonOrLink(e.target as Element) && thread.annotation) {
+        if (
+          !isFromChildButtonOrLink(e.target as Element) &&
+          thread.annotation
+        ) {
           scrollToAnnotation(thread.annotation);
         }
       }}
       onMouseEnter={() => setThreadHovered(thread.annotation ?? null)}
       onMouseLeave={() => setThreadHovered(null)}
       key={thread.id}
```

Comment:
I haven't been able to tell exactly why we had the `key={thread.id}` prop in the `Card`, but I have left it in the top level component.